src/main/app/controller/PostController.ts

```diff
@@ -39,6 +45,23 @@ export class PostController<T extends AnyObject> {
       res.redirect(nextUrl);
     });
   }
+
+  private saveAndSignOut(req: AppRequest<T>, res: Response) {
```

Comment:
perhaps save and signout should have it's own post controller?

Reply:
👍 Good idea, I've refactored this out into its own post controller in 168f2d0